scripts/jerigon_zero_benchmark.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Lets stop using bash scripts

Reply:
alternative for it?

Reply:
https://github.com/0xPolygonZero/zk_evm/issues/774